src/crewai/crew.py

```diff
@@ -295,6 +297,29 @@ def validate_end_with_at_most_one_async_task(self):
 
         return self
 
+    @model_validator(mode="after")
```

Comment:
Is it okay to chain multiple conditionals back to back?

Reply:
why not ?